packages/utility-features/kube-api/src/endpoints/index.ts

```diff
@@ -48,3 +48,4 @@ export * from "./service-account.api";
 export * from "./stateful-set.api";
 export * from "./storage-class.api";
 export * from "./vertical-pod-autoscaler.api";
+export { podMetricsApiInjectionToken } from "./pod-metrics-api-injection-token";
```

Comment:
This is not the place for this, see my PR #7792

Reply:
Thanks, rebased to get #7792 and while resolving conflicts also did this move